src/containers/AccountPageContainer.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { toggleSetting } from '../actions/actions';
+import Setting from '../components/Setting';
+
+const AccountPage = ({ settings, onSettingToggle }) => (
+  <ul>
+    {settings.map(setting =>
+      <Setting
+        name={setting.name}
+        key={setting.name}
+        {...setting}
+        onClick={() => onSettingToggle(setting.name)}
+      />,
+    )}
+  </ul>
+);
+
+AccountPage.propTypes = {
+  settings: PropTypes.arrayOf(PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    checked: PropTypes.bool.isRequired,
+  }).isRequired).isRequired,
+  onSettingToggle: PropTypes.func.isRequired,
+};
```

Comment:
Put the things above this line in a separate `AccountPage.jsx` file under the components directory, this is not part of the container.